src/sentry/dynamic_sampling/latest_release_booster.py

```diff
@@ -161,3 +164,50 @@ def add_boosted_release(project_id: int, release_id: int, environment: Optional[
         datetime.utcnow().replace(tzinfo=UTC).timestamp(),
     )
     redis_client.pexpire(cache_key, BOOSTED_RELEASE_TIMEOUT * 1000)
+
+
+@dataclass(frozen=True)
+class BoostedRelease:
+    version: str
+    environment: Optional[str]
+    platform: Platform
+    timestamp: float
+
+
+def _get_project_platform_from_release(release: Release, project_id: int) -> Optional[Project]:
+    try:
+        return release.projects.get(id=project_id)
+    except Project.DoesNotExist:
+        # If we don't find the project of this release we just default to having no platform name in the BoostedRelease.
+        pass
+
+
+def get_boosted_releases_augmented(project_id: int, limit: int) -> List[BoostedRelease]:
+    """
+    Returns a list of boosted releases augmented with additional information such as release version and platform.
+    """
+    cached_boosted_releases = get_boosted_releases(project_id)
```

Comment:
My assumption we can return `BostedReleases` type, so we can use some new methods